### DISCUSSION TITLE
🎨 Project designs / Diseños del proyecto 🎨

### DISCUSSION THREAD

**Discussion by Omitg24** [Arquisoft/lomap_es3a#35] [Ideas]
<p align="center">
<img src="https://user-images.githubusercontent.com/91057639/218811134-2eec75dc-a6d5-42f7-b1d6-fae6bd5ac158.png" width="640" height="320">
</p>

# EN: Discussion objective 
<p align="justify">
Here we will offer ideas about the graphical aspects of the project (front-end), showing what we think about specific designs or what we would like to see.
</p>

# ES: Objetivo de la discusión
<p align="justify">
Aquí ofreceremos ideas sobre los aspectos gráficos del proyecto (front-end), mostrando lo que pensamos acerca de diseños específicos o aquello que nos gustaría ver.
</p>

**Comment by Omitg24** [comment]
# Subida de diseños
<p align="justify">
Hola equipo, a pesar de que ya os los he enseñado en privado a cada uno, os comparto los diseños hechos (hasta el momento) para el programa, si hay algo que mejoraríais, ya sea ahora o para cualquier futuro diseño comentadlo!
</p>

## Símbolo del proyecto
<p align="center">
<img src="https://user-images.githubusercontent.com/91057639/218886401-607f3573-9b63-4f2f-8f21-cc3fcfe30e3c.png">
</p>

## Logo base del proyecto
<p align="center">
<img src="https://user-images.githubusercontent.com/91057639/218886399-54e41788-9464-49d2-a785-a62f6d74aa05.png">
</p>

## Logo completo (con equipo desarrollador) del proyecto
<p align="center">
<img src="https://user-images.githubusercontent.com/91057639/218886398-df07a4bf-cd45-4b9a-b5fc-86e0a1439958.png">
</p>

## Wallpaper del proyecto
<p align="center">
<img src="https://user-images.githubusercontent.com/91057639/218886402-4f52c78f-f548-4b96-b086-12bd7a3a852a.png">
</p>

Muchas gracias,
Omar

**Reply by uo284373** [reply]
Hola Omar, en mi opinión tanto el logo completo como el Wallpaper están muy bien desarrollados. No haría ningún cambio.

Un saludo.
Carlos

**Reply by Omitg24** [reply]
Perfecto gracias!

Un saludo,
Omar

**Reply by raulEspaa** [reply]
Opino lo mismo que Carlos muy buen diseño, representa muy bien la idea.

Un saludo,
Raúl

**Reply by Nihilhatw** [reply]
Es un gran diseño, me parece muy buena imagen del proyecto los elementos usados en el mismo, describen de forma clara y sencilla la intención de la aplicación.

Un saludo,
David

**Comment by Omitg24** [comment]
# Actualización del Símbolo.
He actualizado el simbolo del proyecto, es un cambio mínimo pero que hace que quede algo mejor (en mi opinión), además, respeta más las convenciones de los marcadores.

<p align="center">
<img src="https://user-images.githubusercontent.com/91057639/221427514-b92fc202-00e2-42e1-8b6e-33b2f719e8db.png">
</p>

**Comment by Omitg24** [comment]
# Diseños de pantalla.
Hola equipo!
He subido los diseños de pantalla a la documentación, ¿que os parecen?
<p allign="center">
<img src="https://user-images.githubusercontent.com/91057639/223270770-c1321590-8757-4db6-8db7-dc0058243385.png" />
</p>
<p allign="center">
<img src="https://user-images.githubusercontent.com/91057639/223270787-b104dc5a-9da2-4037-980b-9875b8f45de5.png" />
</p>
<p allign="center">
<img src="https://user-images.githubusercontent.com/91057639/223270819-4aae5b5b-5b24-455c-886d-a81300d6f012.png" />
</p>

**Reply by raulEspaa** [reply]
Me parecen muy buenos diseños, yo en el ultimo veo un poco mal los nombres pero entiendo que es por la imagen

Un saludo,
Raúl.

**Reply by uo284373** [reply]
Hola, Omar:

En mi opinión, están muy bien desarrollados estos diseños, ya que cumplen con todas las especificaciones del proyecto. Lo único que modificaría sería en el último diseño que pondría en inglés la palabra "Valoración" y "Marcadores".

Un saludo y gracias.
Carlos

**Comment by Omitg24** [comment]
# Nuevos diseños y aviso!
<p align="justify">
Hola equipo, a pesar de que he realizado nuevos diseños (los cuales ya habéis visto), no los subí por que se me pasó básicamente, por lo que, como tendré que actualizarlos de cara a las nuevas funciones que se incluirán y los cambios que han ido surgiendo los subiré aquí, no obstante, os dejo la versión actual.
</p>

![HomeView](https://user-images.githubusercontent.com/91057639/230963801-e3f7fa79-0872-4c97-ac3c-dd7437f551ad.png)
![MapView](https://user-images.githubusercontent.com/91057639/230963810-59039688-d947-41df-9f9e-74c65cca06a6.png)
![MapViewWithOptions](https://user-images.githubusercontent.com/91057639/230963852-d5cd6e7f-db53-47fd-b911-7186ca2c937f.png)
![MapViewAddMarker](https://user-images.githubusercontent.com/91057639/230963828-edb8d734-1150-4282-b34d-c07d07ef515b.png)
![MapViewShowMarker](https://user-images.githubusercontent.com/91057639/230963838-f660ef05-6ecc-4466-9359-a0809f27f536.png)

**Comment by Omitg24** [comment]
# Diseños finales!

Hola equipo, os adjunto los diseños finales de la aplicación:

![HomeView](https://user-images.githubusercontent.com/91057639/235767502-a1c57b84-ad68-42fa-b093-9b296bb191d0.png)
![MapView](https://user-images.githubusercontent.com/91057639/235767537-de7e0426-265e-46e4-96c6-370d811ca930.png)
![MapViewAddMarker](https://user-images.githubusercontent.com/91057639/235767552-8b5b5aa9-e33e-4ed3-90ba-9ae64f888635.png)
![MapViewShowMarker](https://user-images.githubusercontent.com/91057639/235767563-d1df2c89-c39a-48d4-bd1a-f46855ab1c9f.png)
![HelpView](https://user-images.githubusercontent.com/91057639/235767586-863bafd5-bf1b-4e10-850b-88ec54c0d806.png)
![AboutView](https://user-images.githubusercontent.com/91057639/235767633-93983f84-76cb-4d4b-8113-25e4cadd89d8.png)
